app/src/main/java/ch/epfl/sdp/appart/user/User.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sdp.appart.user;
+
+public interface User {
+
+    /**
+     * @return the name of the user
+     */
+    public String getName();
+
+    /**
+     * sets the name of the user
+     */
+    public void setName();
+
+    /**
+     * getter method for ID
+     * @return the unique id of the user
+     */
+     public String getUserId();
+
+    /**
+     * setter method for user email
+     */
+    public void setUserEmail();
+
+    /**
+     * getter method for user email
+     * @return user email as a string
+     */
+     public String getUserEmail();
+
+    /**
+     * setter method for user phone number
+     */
+    public void setPhoneNumber();
+
+    /**
+     * getter method for user phone number
+     * @return user phone number as a string
+     */
+     public String getPhoneNumber();
+
+    /**
+     * setter method for user profile picture
+     */
+     public void setProfileImage();
+
+    /**
+     * getter method for user profile picture
+     * @return the profile picture (Object type was chosen as generic, should be changed once the object Picture is identified)
+     */
+    public Object getProfileImage();
+
+    /**
+     * send a verification email to the user
+     * @return true if email has been correctly sent, false otherwise
+     */
+    public boolean sendEmailVerification();
```

Comment:
I think this should be a Login service method that takes a user as an argument. I think the user shouldn't send itself an email verification.

Reply:
Absolutely!